### ISSUE TITLE
0.17.0 is not installable with yarn2 due to invalid requirement

### ISSUE THREAD

**Issue by Neo-Oli** [prettier/plugin-php#1754]
Hey.

This package is no longer installable with yarn2 due to an invalid requirement. Classic yarn has apparently no problems with this, but yarn2 will throw an error for the `php-parser` requirement. Previous versions worked without a problem. 

```
~/test $ yarn set version berry; yarn init -y; yarn add @prettier/plugin-php
➤ YN0000: Downloading https://github.com/yarnpkg/berry/raw/master/packages/yarnpkg-cli/bin/yarn.js
➤ YN0000: Saving the new release in .yarn/releases/yarn-2.4.2.cjs
➤ YN0000: Done in 2s 147ms
{
  name: 'test'
}
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: php-parser@git+https://github.com/glayzzle/php-parser.git#bb89fff: Couldn't resolve "bb89fff" as either a commit, a tag, or a head - if a commit, use the 40-characters commit hash
```

Probably related to 109db11bf1f335c4fcdb97239fe5558110429407
https://github.com/prettier/plugin-php/blob/95209e144359b3f51e84bcfcef1df6927973d406/package.json#L16

**Comment by czosel** [comment]
Hi @Neo-Oli, thanks for the bug report. The error message is helpful: "use the 40-characters commit hash". Fix is coming up in #1755.

**Comment by czosel** [comment]
Just released 0.17.1 with the fix :tada:
